### ISSUE TITLE
Focus jumps in search edit in Outlook 2016 when navigating on inbox

### ISSUE THREAD

**Issue by iamjett27** [nvaccess/nvda#6681]
Hi, 

May I escalate an NVDA problem in Outlook? One of my colleague is browsing her e-mails in Outlook when suddenly her focus moves to the “Search” fields in Outlook and she is only pressing arrow keys in navigating her email. We really appreciate your feedback and assistance regarding this matter. Thank you and have a wonderful day.

She is using windows 10 and office 365 2016 and new version on nvda

**Comment by josephsl** [comment]
Hi, can you ask your colleague if she is aware of using any add-ons, and if so, try using NVDA with add-ons disabled to see if an add-on or something may have caused this? Thanks.

**Comment by iamjett27** [comment]
Thanks, will check this and get back to you.

**Comment by iamjett27** [comment]
Hi again, as we checked she doesn't have any add-ons and we have also re-installed her NVDA. But still same issue in Outlook.

**Comment by josephsl** [comment]
Hi, Okay, can you ask her steps she took to arrive at this problem? If I or others can reproduce this, I’ll ask for more info. Thanks.

**Comment by iamjett27** [comment]
Hi. according to her, this issue started when Yahoo servers went down and when server issues resolved.

**Comment by josephsl** [comment]
Hi, Does this happen with other accounts? If not, then there’s nothing we can really do (or at least I cannot think of other things to try at this time) except asking for input/output log to find out what’s going on. Thanks.

**Comment by ehollig** [comment]
@iamjett27 is this still happening? If so, we may want a [debug log](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) demonstrating the issue.

**Comment by PratikP1** [comment]
I have been using Outlook 2016 with multiple versions of NVDA. I have never encountered this issue. A debug log is indeed necessary.

**Comment by ehollig** [comment]
Unfortunately, we have not heard from the original author in over 2 years and still require information in order to further debug this issue. Closing as invalid
